lib/state/ui/reducer.ts

```diff
@@ -21,17 +20,41 @@ const simperiumConnected: A.Reducer<boolean> = (state = false, action) =>
     ? action.simperiumConnected
     : state;
 
-const visiblePanes: A.Reducer<string[]> = (
+const visiblePanes: A.Reducer<Set<string>> = (
   state = defaultVisiblePanes,
   action
 ) => {
-  if ('TAG_DRAWER_TOGGLE' === action.type) {
-    return action.show
-      ? union(state, ['tagDrawer'])
-      : difference(state, ['tagDrawer']);
+  switch (action.type) {
+    case 'App.selectTrash': {
+      return new Set(state).add('trash');
+    }
```

Comment:
we only need the curlies if we want to create a new scope/if we want to create variables. they won't be necessary here.